src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -86,6 +85,13 @@ struct TraceSpacetimeChristoffelFirstKind : db::SimpleTag {
   }
 };
 template <size_t Dim, typename Frame, typename DataType>
+struct TraceSpatialChristoffelFirstKind : db::SimpleTag {
```

Comment:
All these simple tags should be tested in the future.

Reply:
.. and opened #1493